docs/integrations/databases/opentelemetry/memcached-opentelemetry.md

```diff
@@ -0,0 +1,155 @@
+
+<img src={useBaseUrl('img/integrations/databases/memcached.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The [Memcached](https://memcached.org/about) app is a logs based app that helps you monitor your Memcached clusters. Preconfigured dashboards provide insight into errors, warnings, and commands executed.
+
+The Sumo Logic App for Memcached is tested for Version: 1.4.15.
+
+Memcache logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Memcached-OpenTelemetry/Memcached-Schematics.png')} alt="Schematics" />
+
+## Sample Log Message
+
+```
+Jun 23 07:35:01 node03 memcached: \
+<31 set GFcIh47CswfCnwk3JkmJ 0 0 4096
+```
+
+## Sample Query
+
+Following is the query from Errors panel of Memcached app's overview Dashboard :
+
+```sql
+%"deployment.environment"=* %"db.cluster.name"=* %"sumo.datasource"=memcached memcached ">" ERROR | json "log" as _rawlog nodrop 
+| if (isEmpty(_rawlog), _raw, _rawlog) as memcached_log_message
+| parse regex field=memcached_log_message ">(?<pid>\d+) (?<cmd>\w+)"
+| if(cmd matches "ERROR",1,0) as ERROR
+| timeslice by 1h 
+| sum(ERROR) as ERROR by _timeslice
+```
+
+## Fields creation in Sumo Logic for Memcached
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of Memcached App install if not already present.
+
+- **`db.cluster.name`** - User configured. Enter a name to identify this Memcache cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`db.system`** - Has a fixed value of memcached
+- **`deployment.environment`** - User configured. This is the deployment environment where the Memcache cluster resides. For example: dev, prod or qa.
+- **`sumo.datasource`** - Has a fixed value of memcached
+
+## Prerequisites
+
+1.  Configure logging in Memcached : By default, the installation of Memcached will not write any request logs to disk. To add a log file for Memcached, you can use the following syntax:
+    ```sql
+    memcached -d -m 3072 -l localhost -p 11211 -u nobody -v 2>>/var/log/memcached/memcached.log
+    ```
+2.  Or, if you're on RHEL/CentOS, you can edit the file `/etc/sysconfig/memcached` like so:
+    ```sql
+    PORT="11211"
+    USER="memcached"
+    MAXCONN="3048"
+    CACHESIZE="256"
+    OPTIONS="-vv >> /var/log/memcached/memcached.log 2>&1"
+    ```
+3.  Save the file and restart Memcached.
+
+## Configure Memcached Logs Collection and App installation
+
+### Step1: Set up Collector
+
+:::note
+If you want to use an existing OpenTelemetry Collector, you can skip this step by selecting the **Use an existing Collector** option.
+:::
+
+If you want to create a new Collector please select **Add a new Collector** option.
```

Comment:
```suggestion
To create a new Collector:
	
1. Select the **Add a new Collector** option.
2. Select the platform for which you want to install the Sumo OpenTelemetry Collector.
	
This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
```